src/VisualStudio/Core/Def/Implementation/AddUsingsOnPaste/VisualStudioAddUsingsOnPaste.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable enable 
+
+using System;
+using System.Threading.Tasks;
+using System.Composition;
+using Microsoft.CodeAnalysis.Editor.Shared.Utilities;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+using Microsoft.CodeAnalysis.Text;
+using System.Threading;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.CodeAnalysis.Host;
+using Microsoft.CodeAnalysis.Internal.Log;
+using Microsoft.VisualStudio.Utilities;
+using System.Diagnostics;
+using Microsoft.CodeAnalysis.Editor.AddMissingImports;
+using Microsoft.CodeAnalysis.AddMissingImports;
+
+namespace Microsoft.VisualStudio.LanguageServices.Implementation.AddUsingsOnPaste
+{
+    [ExportWorkspaceService(typeof(IAutomaticallyAddMissingUsingsService)), Shared]
+    internal class VisualStudioAddUsingsOnPaste : IAutomaticallyAddMissingUsingsService
```

Comment:
Nit: Seems like we use the term "Imports" and not "usings" in the code, so should this be renamed to match?

Reply:
Good question. Let me see if the verbage is different between C# and VB or we need language specific.

Reply:
Go with Import as the general term.